### ISSUE TITLE
Encode picture

### ISSUE THREAD

**Issue by Nehved** [song940/node-escpos#245]
Hi there. I have some troubles with printing images. Here my source code
`const device = new escpos.Network(IP);
const options = { encoding: 'CP860' };
const printer = new escpos.Printer(device, options);
const tux = path.join(__dirname, 'tux.png');

escpos.Image.load(tux, (image) => {
  device.open(() => {
    printer.text('hello here');
    printer.image(image);
    printer.cut();
    printer.close();
  });
});`

below I attached photo of result, thanks for your help)

![photo_2019-12-17_18-02-05](https://user-images.githubusercontent.com/26201207/71007559-3f0cc080-20f8-11ea-98a8-28ffbbe001e8.jpg)

**Comment by Nehved** [comment]
Btw, I use Ubuntu 18.04 + Electron app, printer HP.

**Comment by Psychopoulet** [comment]
printer.image is an async method

**Comment by Nehved** [comment]
@Psychopoulet thanks for response, I refactored my code in next way: 
```
const { Printer, Network, Image } = require('escpos');
const device = new Network(ip);
const printer = new Printer(device);
const tux = path.join(__dirname, 'tux.png');

Image.load(tux, (image) => {
  device.open(async () => {
    await printer.image(image);
    printer.cut().close();
  });
});
```
result still the same, could you pls provide working example?

**Comment by Psychopoulet** [comment]
... in your electron app do you use this package in a front way ??

**Comment by Nehved** [comment]
@Psychopoulet no, it is located in my main file for backend part, like "electron.js"

**Comment by Psychopoulet** [comment]
what is your printer's model ?
are you sure thate escpos standard is enable on it ?
